sdk/test/utxo.test.ts

```diff
@@ -435,36 +441,4 @@ describe('UTXO Tests', () => {
         assert(zeroBalance.unconfirmed === 0n, 'If no address specified unconfirmed must be 0');
         assert(zeroBalance.total === 0n, 'If no address specified total must be 0');
     });
-
-    it('returns smalled amount if address holds ordinals', async () => {
```

Comment:
Why did you remove this test case?

Reply:
this test will fail because in this one we were checking that half of the outputs were ordinals.
because now there's a follow-up check with inputs to that output containing ordinals it's no longer valid.
I'll restore it if I find a way to mock many more apis i.e. `getTransactionHex` etc

Reply:
upd: updated test

with this change we can assume that `confirmed` balance value will be less that `confirmed` total amount from ord service but `total`s must be equal